rules/windows/image_load/image_load_dll_taskschd_load_from_suspicious_process.yml

```diff
@@ -0,0 +1,33 @@
+title: Suspicious Task Scheduler DLL Load
+id: 3b92a1d0-8d4b-4d28-a1b4-1e29d49a6a3e
+status: experimental
+description: |
+  Detects the loading of the taskschd.dll module from suspicious processes from uncommon directories, which could indicate a scheduled task creation attempt via COM object (Schedule.Service) abuse.
+references:
+    - https://www.logpoint.com/en/blog/shenanigans-of-scheduled-tasks/
+    - https://x.com/Max_Mal_/status/1826179497084739829
+author: Swachchhanda Shrawan Poudel
+date: 2024-08-26
+tags:
+    - attack.persistence
+    - attack.execution
+    - attack.t1053.005
+logsource:
+    category: image_load
+    product: windows
+detection:
+    selection:
+        - ImageLoaded|endswith: 'taskschd.dll'
+        - OriginalFileName: 'taskschd.dll'
+    filter:
+        Image|contains:
+            - '\AppData\Local\Temp\'
+            - 'C:\Users\Public\'
+            - 'C:\Windows\Temp\'
+            - 'C:\Temp\'
+            - '\Downloads\'
+            - '\Desktop\'
+    condition: selection and not filter
```

Comment:
I think the logic of the rule is incorrect as probably you wanted to include these paths.

But since these paths are also installers paths too. This could be very prone to FP depending on the env and apps. I'll be moving this to TH folder

Reply:
You can see this in action

- https://github.com/SigmaHQ/sigma/actions/runs/10671264627/job/29576731247?pr=4983
- https://github.com/SigmaHQ/sigma/actions/runs/10671264627/job/29576731515?pr=4983
- https://github.com/SigmaHQ/sigma/actions/runs/10671264627/job/29576732058?pr=4983
- https://github.com/SigmaHQ/sigma/actions/runs/10671264627/job/29576732510?pr=4983